src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectBindPanel.java

```diff
@@ -206,15 +227,30 @@ private void setProjects() {
       }
     } else {
       projectList.setModel(new DefaultListModel<>());
-    }
-
-    // it can be happen because server has no projects, no server selected, or server not updated
-    if (projectList.isEmpty()) {
       projectList.setEmptyText(getProjectEmptyText());
       projectList.setEnabled(false);
     }
   }
 
+  /**
+   * Assumes that it's bound and a server is selected
+   */
+  private void downloadProjectList() {
+    ApplicationManager.getApplication().assertIsDispatchThread();
+
+    SonarQubeServer selectedServer = getSelectedServer();
+    ServerDownloadProjectTask downloadTask = new ServerDownloadProjectTask(project, engine, selectedServer);
+
+    try {
+      downloadTask.queue();
+      Map<String, RemoteModule> map = downloadTask.getResult();
+      setProjectsInList(map.values());
+    } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)